cmd/crc/cmd/daemon_unix.go

```diff
@@ -0,0 +1,21 @@
+// +build !windows
+
+package cmd
+
+import (
+	"net"
+	"os"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+)
+
+func httpListener() (net.Listener, error) {
+	_ = os.Remove(constants.DaemonHTTPSocketPath)
+	ln, err := net.Listen("unix", constants.DaemonHTTPSocketPath)
+	logging.Infof("listening %s", constants.DaemonHTTPSocketPath)
+	if err != nil {
```

Comment:
Inconsistent with how `vsocklistener` is written. there seems to be more logging here.

Reply:
done